### PR TITLE
feat: dart shooter balance experiment

### PR DESCRIPTION
**Pull request by OzPayn**
reduced firing speed
slightly reduced projectile speed
reduced damage
reduced power enchantment modifier

### CODE REVIEW

File: src/main/java/com/aetherteam/aether/entity/projectile/dart/EnchantedDart.java
Comment (Sto3IV) [inline src/main/java/com/aetherteam/aether/entity/projectile/dart/EnchantedDart.java:1]: from 6.0 to 4.0 will be more fine

File: src/main/java/com/aetherteam/aether/entity/projectile/dart/GoldenDart.java
Comment (Sto3IV) [inline src/main/java/com/aetherteam/aether/entity/projectile/dart/GoldenDart.java:1]: no

File: src/main/java/com/aetherteam/aether/entity/projectile/dart/PoisonDart.java
Comment (Sto3IV) [inline src/main/java/com/aetherteam/aether/entity/projectile/dart/PoisonDart.java:1]: no. current numbers if fine

File: src/main/java/com/aetherteam/aether/item/combat/DartShooterItem.java
Comment (Sto3IV) [inline src/main/java/com/aetherteam/aether/item/combat/DartShooterItem.java:1]: depends on Minecraft difficulty will be better.

**Review by Sto3IV — CHANGES_REQUESTED** [review]
easy difficulty = ez mode
hard difficulty = please no. no darts.... i'm falling. ded

**Comment by OzPayn** [comment]
> easy difficulty = ez mode hard difficulty = please no. no darts.... i'm falling. ded

Difficulty settings don't generally affect player functionality in vanilla, I do not think that would be a good idea.

The damage numbers Ive chosen are based on the amount of hits required to kill some of our tankier mobs like Valkyries. If the value is too high the dart shooters very very quickly outcompete even the highest enchanted swords. The values I've proposed here put the dart shooters just below swords in terms of damage output, but they also still retain the very impactful strength of being zero-gravity projectiles. So I think there would be an argument for perhaps debuffing them further. But I'd like people to actually test these values first and see how they feel rather than just commenting on the numbers themselves.

**Review by bconlon1 — APPROVED** [review]
lgtm

**Pull request merged**